### PR TITLE
Make Poisonwood's spaceport change in response to the path taken during FW Alphas

### PR DESCRIPTION
**Pull request by Anarchist2**
## Summary
This PR adds two new Poisonwood spaceport descriptions, with one for teaming up with the Oathkeepers and one for when your karma is too low. These descriptions reflect the events that occur in each path.

### CODE REVIEW

**Comment by Zitchas** [comment]
I like the idea of having planet descriptions change to reflect major events in the storyline. Makes the world feel more alive.

**Review by roadrunner56 — APPROVED** [review]
I really appreciate it when video games have little details like this that show that you have an effect on the world.

File: data/human/events.txt [hunk 1]
```diff
@@ -1392,6 +1392,9 @@ event "liberation of Poisonwood"
 		fleet "Small Free Worlds" 700
 		fleet "Small Free Worlds" 1200
 		fleet "Human Miners" 2000
+	planet Poisonwood
+		spaceport `The starport is a hodgepodge of small metal structures, built from the remains of Navy and pirate ships that fought for control of this world. The locals here welcome you to their planet heartily.`
+		spaceport `	The tavern in the center of the port is named The Parting Glass. The walls are covered with photographs from after the war: liberated slaves boarding Navy ships to leave this planet that had brought them so much pain; Navy officers leaving their ships to retire on this world that they liberated; friends gone away, and friends left behind. There are also newer photos mixed in amongst the older ones: two Falcons and a Navy Cruiser landed side by side; troops clad in armor walking away from a factory, and a familiar captain talking to a man you know as Commander Nguyen. It is a strange feeling to be shown along heroes of a history that you have had only the smallest effect on.`
```
Comment (Pointedstick) [inline data/human/events.txt:1397]:
```suggestion
		spaceport `	The tavern in the center of the port is named The Parting Glass. Its walls are covered with photographs from after the war: liberated slaves boarding Navy ships to leave this planet that had brought them so much pain; Navy officers leaving their ships to retire on this world that they liberated; friends gone away; friends left behind. There are also newer photos mixed in amongst the older ones: two Falcons and a Navy Cruiser landed side by side; troops clad in armor walking away from a factory, and a familiar captain talking to a man you know as Commander Nguyen. It is a strange feeling to be shown along heroes of a history that you have had only the smallest effect on.`
```

(too many uses of the word "the" in close proximity)

File: data/human/events.txt [hunk 2]
```diff
@@ -1402,6 +1405,9 @@ event "Poisonwood reverts to Republic"
 		fleet "Small Free Worlds" 1500
 		fleet "Small Free Worlds" 2600
 		fleet "Human Miners" 2000
+	planet Poisonwood
+		spaceport `The starport is a hodgepodge of small metal structures, built from the remains of Navy and pirate ships that fought for control of this world. The locals are unusually courteous and proud to most of the captains landing here, but not to you.`
```
Comment (Pointedstick) [inline data/human/events.txt:1409]:
```suggestion
		spaceport `The starport is a hodgepodge of small metal structures, built from the remains of Navy and pirate ships that fought for control of this world. The locals are unusually courteous and friendly to most of the captains landing here, but not to you.`
```

**Review by Pointedstick — COMMENTED** [review]
Cool stuff, I like this kind of dynamic response to in-game events too. Just a few suggestions.

File: data/human/events.txt (hunk 1 above)
Comment (MasterOfGrey) [reply data/human/events.txt:1397]: Should also probably be "shown _among_ heroes".

Comment (Pointedstick) [reply data/human/events.txt:1397]: Indeed, that too.

**Review by Pointedstick — APPROVED** [review]

**Pull request merged**